blueocean-dashboard/src/main/js/components/karaoke/components/Pipeline.jsx

```diff
@@ -271,7 +271,7 @@ export default class Pipeline extends Component {
                 nextPath = `/${pathArray.join('/')}`;
             }
             // check whether we have a parallel node or seq parallel nodes
-            const isParallel = nextNode.isParallel || (nextNode.firstParent && nextNode.parent != nextNode.firstParent);
+            const isParallel = nextNode.isParallel || nextNode.isSequential;
```

Comment:
Minor nit, isParallel is now true if its parallel or sequential.

Maybe it should now be isTopLevelNode?

Reply:
Only the first stage in each parallel branch is marked `isParallel` - I didn't wanna change it because who knows what will break where.

Reply:
well i meant more the variable

`const isParallel` => `const isTopLevelNode` or something

Reply:
That's a good idea, I'll do that. I don't wanna change anything serious in this Rube Goldberg machine, but anything we can do to increase readability without damage gets my vote.